pkg/flowkit/arguments_test.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 func TestArguments(t *testing.T) {
-
+	cadenceString, _ := cadence.NewString("42")
```

Comment:
I these cases, I think you can just use `cadence.String("42")` which doesn't return an error. Which means there is no need to pull it out.

Reply:
🤦  thank you